src/test/java/com/ezylang/evalex/functions/datetime/DateTimeFunctionsTest.java

```diff
@@ -231,6 +241,24 @@ void testDateTimeToEpoch(String expression, String expectedResult)
     assertExpressionHasExpectedResult(expression, expectedResult);
   }
 
+  @ParameterizedTest
```

Comment:
I'd like to have separate test functions for `DT_DATE_NOW` and `DT_TODAY`.
An idea how to check the current time correctness, without changing system defaults is this approach:
```
import java.time.Instant;
import java.time.temporal.ChronoUnit;
import org.assertj.core.api.Assertions;

public class TimeTest {
    @Test
    public void testCurrentTime() {
        Instant expectedTime = Instant.now();
        Instant actualTime = Instant.now(); // Replace this with the method that returns the current time

        // Truncate both times to seconds to avoid failing due to milliseconds difference
        expectedTime = expectedTime.truncatedTo(ChronoUnit.SECONDS);
        actualTime = actualTime.truncatedTo(ChronoUnit.SECONDS);

        Assertions.assertThat(actualTime).isEqualTo(expectedTime);
    }
}
```